### PR TITLE
dma_logee_driver to airflow

### PR DESCRIPTION
**Pull request by RendiKustiawan**

### CODE REVIEW

**Review by ameliaksalwa — CHANGES_REQUESTED** [review]
punteun edit commitnya bukan di `bq_view`, tapi `dags -> source -> sql -> dwh -> L1 -> visibility`

File: dags/config/L1/visibility/dma_logee_driver.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+dag_id: L1_visibility_dma_logee_driver
+source: logee-data-prod.logee_datalake_raw_production.visibility_dma_logee_driver
+destination: logee-data-prod.L1_visibility.dma_logee_driver
+sql: source/sql/dwh/L1/visibility/lgd_dma_logee_driver.sql
+schedule: 0 */3 * * *
+time_partitioning: published_timestamp
```
Comment (fikrihaykal41) [inline dags/config/L1/visibility/dma_logee_driver.yaml:6]: soalnya dipake disini kolom `published_timestamp`-nya

File: dags/source/sql/dwh/L1/visibility/dma_logee_drivers.sql [hunk 2]
```diff
@@ -23,7 +25,6 @@ SELECT
      DATE(IF(REPLACE(JSON_EXTRACT(data, '$.simExpiryDate'), '"', '') = '', NULL, REPLACE(JSON_EXTRACT(data, '$.simExpiryDate'), '"', ''))),
      NULL 
   ) AS sim_expired,
-  data AS original_data,
   ts AS published_timestamp
```
Comment (fikrihaykal41) [inline dags/source/sql/dwh/L1/visibility/dma_logee_drivers.sql:28]: kayanya `published_timestamp` tipe datanya masih `STRING` deh, belum `TIMESTAMP`

**Review by fikrihaykal41 — CHANGES_REQUESTED** [review]
punteun

**Comment by toileto** [comment]
Branch is taken over from Rendi. New PR is available here yaa: https://github.com/Logee-Data/logee-dwh/pull/75

**Pull request closed without merging**